internal/test/oats/sql/yaml/oats_sql_go_postgres.yaml

```diff
@@ -16,11 +16,13 @@ expected:
             db.collection.name: accounting.contacts
             db.system.name: other_sql
   metrics:
-    - promql: 'db_client_operation_duration_sum{db_operation_name="SELECT", db_system_name="other_sql"}'
+    - promql: 'db_client_operation_duration_sum{db_operation_name="SELECT", db_system_name="other_sql", server_address="sqlserver"}'
```

Comment:
This is great test, but I think we need to add one for mysql and the alternative postgres library you added.